### ISSUE TITLE
[Meson] Fails to auto-start via DBus activation due to incomplete path

### ISSUE THREAD

**Issue by jbeich** [hyprwm/xdg-desktop-portal-hyprland#160]
Regressed by #154 due to placeholder rename: `@libexecdir@` -> `@LIBEXECDIR@`.
```
$ meson setup --prefix=/tmp/xdg-desktop-portal-hyprland_prefix /tmp/xdg-desktop-portal-hyprland_build
[...]
meson.build:54: WARNING: The variable(s) 'LIBEXECDIR' in the input file 'org.freedesktop.impl.portal.desktop.hyprland.service.in' are not present in the given configuration data.
[...]
$ meson compile -C /tmp/xdg-desktop-portal-hyprland_build
$ meson install -C /tmp/xdg-desktop-portal-hyprland_build
$ fgrep Exec /tmp/xdg-desktop-portal-hyprland_prefix/share/dbus-1/services/org.freedesktop.impl.portal.desktop.hyprland.service
Exec=/xdg-desktop-portal-hyprland
```
Also, `@systemd_service@` placeholder is no longer used:
https://github.com/hyprwm/xdg-desktop-portal-hyprland/blob/86a94c3e4edb39ccc5584eac51a87930593daf2c/meson.build#L35-L37

**Comment by jbeich** [comment]
Also, `systemd_service` placeholder can probably be dropped.
https://github.com/hyprwm/xdg-desktop-portal-hyprland/blob/86a94c3e4edb39ccc5584eac51a87930593daf2c/meson.build#L35-L37

**Comment by fufexan** [comment]
Fixed in 379b27281e2ce4790709eb35b7faa368c89c9f3d.

**Comment by jbeich** [comment]
I confirm, works fine now.
